.github/workflows/ci.yml

```diff
@@ -10,7 +19,7 @@ jobs:
         node-version: 12
     - uses: actions/setup-go@v2
       with:
-        go-version: 1.13
+        go-version: 1.16
```

Comment:
May I know, why not update to 1.18? 🤓

Reply:
There are some more changes in 1.18 with regards to build tags etc. and I don't want to touch this for now